### ISSUE TITLE
impossible command parameter definition error

### ISSUE THREAD

**Issue by TheLimeGlass** [SkriptLang/Skript#6148]
### Suggestion

Add a warning or better an error stating that the user cannot use another parameter after an optional text parameter. Example
```
command /example [<text>] [<text>] [<player>]
```
As Skript will parse everything in the command as one argument if the player is not online.

### Why?

Skript cannot properly define the difference between the requested types, so Skript should error stating that the user should.ppace the player argument before the text parameters.

### Other

This issue is also present when it comes to string and string as two parameter types in Functions. See any test runner results for a present error representation.

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this suggestion.

**Comment by sovdeeth** [comment]
I really don't think this should error. Users should be responsible for checking the validity of their arguments, so checking if arg-3 is set is a perfectly valid way to handle this. Erroring here would lock users out of existing command structures that they may have been using for a while (and I certainly do), for no real reason in my opinion. A warning may make sense for certain types, like player, where their parsing depends on factors outside of the command, but this certainly isn't the case for all parameters. This warning would also require a config option to switch off, as the user may have many valid reasons for putting a non-text argument after a text argument.

For example, should this have a warning/error?
```
command persona <action:text> [<persona:text>] [to <player>] [with colour <text>]:
```
I have a non-text after a text, but it's hardly in danger of causing parsing issues.

To sum up, I think this is a very small benefit for a lot of arduous effort avoiding edge cases and false positives, plus going as far as to error would start causing breaking changes and limit the functionality of Skript commands for no good reason.
